codebuild/bin/install_default_dependencies.sh

```diff
@@ -60,14 +60,14 @@ fi
 
 # Download and Install AWS-LC
 if [[ "$S2N_LIBCRYPTO" == "awslc" && ! -d "$AWSLC_INSTALL_DIR" ]]; then
-    codebuild/bin/install_awslc.sh "$(mktemp -d)" "$AWSLC_INSTALL_DIR" "0" > /dev/null ;
+    codebuild/bin/install_awslc.sh "$(mktemp -d)" "$AWSLC_INSTALL_DIR" > /dev/null ;
 fi
 
-if [[ "$S2N_LIBCRYPTO" == "awslc-fips" && ! -d "$AWSLC_FIPS_INSTALL_DIR" ]]; then
```

Comment:
Should `AWSLC_FIPS_INSTALL_DIR` be removed from s2n_setup_env? Is there anything else that should be removed now that we're not installing the 2021 aws-lc-fips? Do we still have any codebuild jobs using this version of aws-lc-fips?

Reply:
Yeah, I'll clean it up.  There is one dimension in the Integ test I need to also address.